text/0081-flashback.md

```diff
@@ -0,0 +1,104 @@
+# RFC: Return TiKV to some certain version
+
+## Summary
+
+This proposal introduces a method to return TiKV to some certain version, which could be useful when doing some lossy recovery.
+
+## Motivation
+
+The initial motivation is to do lossy recovery on TiKV when deploying TiDB cluster across DCs with DR Auto-sync method.
+
+When use DR Auto-sync method to deploy TiDB cluster in two DC and 
+
+- there is a network isolation between primary and DR DC
+- PD make the replica state of primary DC to be Async after a while
+- the primary DC is down
+
+Then the sync state of the cluster (from the DR DC's PD) is not dependable. In this case, data can be partially sent to the TiKV nodes in DR, and break the ACID consistency.
+
+Though it is impossible to do lossless recovery in this situation, we can recover the data on TiKV nodes to a certain version (`resolved_ts` in this case), which is ACID consistency.
+
+## Detailed design
+
+Basically, we need a method to return TiKV to a certain version (timestamp), we should add a new RPC to kvproto:
+
+```protobuf
+message FlashBackRequest {
```

Comment:
`FlashBack` is usually used to represnet the situation bringing back something dropped unexpectedly, for example a table or a database is dropped mistakenly and the DBMS provides a statement `flaskback table/database` to recover the dropped table or database. Here If it's used to truncate extra data maybe something like `truncate` is better for the name.

Reply:
Here is a flashback [rfc](https://docs.google.com/document/d/1SvNP-vT3CKwhuwAvRrmjFp-Lzdix3c6dMAIHBReGspg/edit#), I think these two are not the same.